src/lib.rs

```diff
@@ -58,17 +58,16 @@
 //!
 //! let bits = 2048;
 //! let private_key = RsaPrivateKey::new(&mut rng, bits).expect("failed to generate a key");
-//! let signing_key = SigningKey::new_with_hash(private_key, Hash::SHA2_256);
-//! let verifying_key: VerifyingKey = (&signing_key).into();
+//! let signing_key = SigningKey::<Sha256>::new_with_hash(private_key, Hash::SHA2_256);
```

Comment:
```rust
SigningKey::<Sha256>::new_with_hash(private_key, Hash::SHA2_256);
```

...still repeats `Sha256` and `Hash::SHA2_256`.

Reply:
This is a bullet 3 in the RFC in the first comment. @sandhose suggested implementing `const_oid::AssociatedOID for the `Digest` implementations instead of having a separate `Hash` class. Is that ok with you?